test/dummy/app/views/layouts/application.html.erb

```diff
@@ -9,7 +9,7 @@
     <%= javascript_importmap_tags %>
   </head>
 
-  <body>
+  <body class="<%= "turbo-native" if turbo_native_app? %>">
```

Comment:
Did you mean to add a test to back this up or is this just a smoke test?

Reply:
My plan was to add a test, but then I chased the CI failures across the other PRs. There will be a test to cover this behavior if we deem it viable.